model-optimizer/mo/utils/version.py

```diff
@@ -21,6 +21,5 @@ def get_version():
     if not os.path.isfile(version_txt):
         return "unknown version"
     with open(version_txt) as f:
-        _ = f.readline()
```

Comment:
We need this function work correctly when there are any number of empty lines.

@artanokhov, @asuhov, @ababushk could you comment when and why the format of version.txt has changed?